task/sast-coverity-check-oci-ta/0.2/sast-coverity-check-oci-ta.yaml

```diff
@@ -733,7 +733,6 @@ spec:
         done
       computeResources:
         limits:
-          cpu: "16"
```

Comment:
Are you sure removing this won't fallback to using the default value? Maybe set by a LimitRange?

Reply:
@sfowl This is exactly what I asked on #1821 when this change was applied to all the other tasks and these were the answers I was given: https://github.com/konflux-ci/build-definitions/pull/1821#discussion_r1916242807

Reply:
nice diagram!